CHANGELOG.md

```diff
@@ -8,10 +8,17 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [2.1.0] - 2019-01-30
+
 ### Added
 
 - Manage xAPI statements in backend application.
 - Listen to plyr player events and send xAPI statements to the backend application.
+- Installation of Grommet in the front application. Will be used to theme our application.
+- Display upload progress when a resource is uploaded.
```

Comment:
```suggestion
- Display upload progress when a video is uploaded.
```

Reply:
I didn't put video because I think the same component is used for every uploads.